__tests__/__integration__/imperative/__tests__/__integration__/cli/config/secure/cli.imperative-test-cli.config.secure.integration.test.ts

```diff
@@ -0,0 +1,200 @@
+*
+*/
+
+import { ITestEnvironment } from "../../../../../../../__src__/environment/doc/response/ITestEnvironment";
+import { SetupTestEnvironment } from "../../../../../../../__src__/environment/SetupTestEnvironment";
+import { runCliScript } from "../../../../../../../src/TestUtil";
+import * as fs from "fs";
+import * as keytar from "keytar";
+import * as path from "path";
+
+
+// Test Environment populated in the beforeAll();
+let TEST_ENVIRONMENT: ITestEnvironment;
+
+describe("imperative-test-cli config secure", () => {
+    let expectedProjectConfigLocation: string;
+    let expectedUserConfigLocation: string;
+    let expectedGlobalProjectConfigLocation: string;
+    let expectedGlobalUserConfigLocation: string;
+    // Create the test environment
+    beforeAll(async () => {
+        TEST_ENVIRONMENT = await SetupTestEnvironment.createTestEnv({
+            cliHomeEnvVar: "IMPERATIVE_TEST_CLI_CLI_HOME",
+            testName: "imperative_test_cli_test_config_secure_command"
+        });
+        expectedGlobalUserConfigLocation = path.join(TEST_ENVIRONMENT.workingDir, "imperative-test-cli.config.user.json");
+        expectedGlobalProjectConfigLocation = path.join(TEST_ENVIRONMENT.workingDir, "imperative-test-cli.config.json");
+        expectedUserConfigLocation = path.join(TEST_ENVIRONMENT.workingDir, "test", "imperative-test-cli.config.user.json");
+        expectedProjectConfigLocation = path.join(TEST_ENVIRONMENT.workingDir, "test", "imperative-test-cli.config.json");
+    });
+    afterEach(() => {
+        runCliScript(__dirname + "/../__scripts__/delete_configs.sh", TEST_ENVIRONMENT.workingDir,
+            ["-rf imperative-test-cli.config.user.json imperative-test-cli.config.json test schema.json"]);
+    });
+    it("should display the help", () => {
+        const response = runCliScript(__dirname + "/../__scripts__/get_help.sh",
+            TEST_ENVIRONMENT.workingDir, ["secure"]);
+        expect(response.output.toString()).toContain(`prompt for secure configuration properties`);
+    });
+    it("should secure the project config", async () => {
+        runCliScript(__dirname + "/../init/__scripts__/init_config_prompt.sh", TEST_ENVIRONMENT.workingDir, [""]);
+        const response = runCliScript(__dirname + "/__scripts__/secure_prompt.sh", TEST_ENVIRONMENT.workingDir, [""]);
+        const fileContents = JSON.parse(fs.readFileSync(expectedProjectConfigLocation).toString());
+        const config = runCliScript(__dirname + "/../list/__scripts__/list_config.sh", TEST_ENVIRONMENT.workingDir, ["--rfj"]).stdout.toString();
+        const configJson = JSON.parse(config);
+        const expectedJson = {
+            defaults: {
+                secured: "my_secured"
+            },
+            profiles: {
+                my_secured: {
+                    type: "secured",
+                    properties: {
+                        info: "",
+                        secret: "anotherFakeValue"
```

Comment:
Should the `secret` property be gone? and `profiles.my_secured.properties === {info: ""}`

Reply:
Same question for a few other tests below

Reply:
I think I got this one confused. I thought I was looking at `set-secure`